bangc-ops/mlu_op.h

```diff
@@ -2154,6 +2154,105 @@ mluOpStatus_t MLUOP_WIN_API mluOpSqrtBackward(
     const mluOpTensorDescriptor_t dy_desc, const void *diff_y,
     const mluOpTensorDescriptor_t dx_desc, void *diff_x);
 
+/*!
+ * @brief Computes bounding box information from the backbone output of the
+ * detected network.
+ *
+ * @param[in] handle
+ *   Input. Handle to a MLUOP context that is used to manage MLU devices and
+ *   queues in the yolo_box operation. For detailed information, see
+ *   ::mluOpHandle_t.
+ * @param[in] x_desc
+ *   Input. The descriptor of the tensors. For detailed information, see
+ *   ::mluOpTensorDescriptor_t.
+ * @param[in] x
+ *   Input. Pointer to the MLU memory that stores the input tensor.
+ * @param[in] img_size_desc
+ *   Input. The descriptor of the tensors. For detailed information, see
+ *   ::mluOpTensorDescriptor_t.
+ * @param[in] img_size
+ *   Input. Pointer to the MLU memory that stores the input tensor.
+ * @param[in] anchors_desc
+ *   Input. The descriptor of the tensors. For detailed information, see
+ *   ::mluOpTensorDescriptor_t.
+ * @param[in] anchors
+ *   Input. Pointer to the MLU memory that stores the input tensor.
+ * @param[in] class_num
+ *    Input. The number of classes.
+ * @param[in] conf_thresh
+ *    Input. The confidence scores threshold of detection boxes.
+ *    Boxes with confidence scores under threshold should be ignored.
```

Comment:
The detection boxes with the confidence score below the threshold should be ignored.